### PR TITLE
V-laanjana/package creation/sophos xg firewall 

### PR DESCRIPTION
**Pull request by v-laanjana**
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution sophos xg
See guidance below
Version Updated:
2.0.0
See guidance below
Testing Completed:
The link for test results is
[Sentinel Solution Migration -TemplateSpec - sophosXGFirewall](https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B3E3CD13F-8052-4464-8F9E-4AF53B257B2C%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20sophosXGFirewall.docx&action=default&mobileredirect=true&share=IQE_0Tw-UoBkRI-eSvU7JXssAcywG_wMpDjgMQaOsxEDZpM&cid=8c5f91ff-7bc7-4bba-8d6d-e6d5d65b7a5c)

See guidance below
This is an ISV solution.

### CODE REVIEW

**Comment by v-amolpatil** [comment]
Please update `isPreview` from `true` to `false` in 

Current code has below values:
"availability": { 
    "status": 1,
    "isPreview": true
    }

File: Solutions/Sophos XG Firewall/Package/mainTemplate.json [hunk 1]
```diff
@@ -0,0 +1,934 @@
...
+      "type": "string",
+      "defaultValue": "Sophos XG Firewall",
+      "minLength": 1,
+      "metadata": {
+        "description": "Name for the workbook"
+      }
+    }
+  },
+  "variables": {
+    "solutionId": "azuresentinel.azure-sentinel-solution-sophosxgfirewall",
```
Comment (v-amolpatil) [inline Solutions/Sophos XG Firewall/Package/mainTemplate.json:41]: Please check if "solutionId" value is correct. 
Should the value of "solutionId" be "azure-sentinel-solution-sophosxgfirewall"?

**Comment by v-amolpatil** [comment]
Please update text `Azure Sentinel` to `Microsoft Sentinel` in `mainTemplate` and `Data Connectors` file.

**Comment by v-amolpatil** [comment]
Please verify if the content of "Data Connectors" file `description` is correct. 

**Currently in code:**
image

**Extected:**
image

File: Solutions/Sophos XG Firewall/Package/createUiDefinition.json [hunk 2]
```diff
@@ -64,35 +95,18 @@
             "name": "workbooks-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This Azure Sentinel Solution installs workbooks. Workbooks provide a flexible canvas for data monitoring, analysis, and the creation of rich visual reports within the Azure portal. They allow you to tap into one or many data sources from Azure Sentinel and combine them into unified interactive experiences.",
+              "text": "This Microsoft Sentinel Solution installs workbooks. Workbooks provide a flexible canvas for data monitoring, analysis, and the creation of rich visual reports within the Azure portal. They allow you to tap into one or many data sources from Microsoft Sentinel and combine them into unified interactive experiences."
```
Comment (v-amolpatil) [inline Solutions/Sophos XG Firewall/Package/createUiDefinition.json:98]: Please verify if workbook content is correct?

In text change document I see below

image

**Review by v-amolpatil — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**